src/pipx/constants.py

```diff
@@ -29,11 +29,21 @@ def is_windows() -> bool:
     return sys.platform == "win32"
 
 
+def is_macos() -> bool:
+    return sys.platform == "darwin"
+
+
+def is_linux() -> bool:
+    return not WINDOWS and not MACOS
```

Comment:
I do think we should make this more explicit.

```suggestion
    return sys.platform == "linux"
```

Reply:
This would not cover a number of cases, see the `sys.platform` docs: https://docs.python.org/3/library/sys.html#sys.platform

Maybe we can call it `is_unix` instead, and keep the logic as is?

Reply:
> Maybe we can call it `is_unix` instead, and keep the logic as is?

Yes, sounds good.

Reply:
My preference would be to use `platform.system() == "Linux"`